PVTPackage/source/MultiphaseSystem/PhaseSplitModel/DeadOilFlash.hpp

```diff
@@ -35,6 +35,20 @@ class DeadOilFlash
                 double waterSurfaceMassDensity,
                 double waterSurfaceMolecularWeight );
 
+  DeadOilFlash( std::vector< std::vector< double > > const & PVDO,
+                double oilSurfaceMassDensity,
+                double oilSurfaceMolecularWeight,
+                std::vector< std::vector< double > > const & PVDG,
+                double gasSurfaceMassDensity,
+                double gasSurfaceMolecularWeight );
+
+  DeadOilFlash( std::vector< std::vector< double > > const & PVDO,
+                double oilSurfaceMassDensity,
+                double oilSurfaceMolecularWeight,
+                std::vector< double > const & PVTW,
+                double waterSurfaceMassDensity,
+                double waterSurfaceMolecularWeight );
```

Comment:
Same here, a `@brief` to tell which combinations we use could be nice.

Reply:
Done, I added the inline Doxygen comments.